tests/installation/validate_beta_popup.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test module to validate beta popup and accept it.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use Test::Assert ':all';
+
+sub run {
+    my $warnings = $testapi::distri->get_warnings_controller();
+    assert_matches(qr/You are accessing our Beta Distribution/, $warnings->get_warning_rich_text(),
+        "Warning Dialog for 'BIOS boot partition has too small size to install bootloader' did not appear, while it is expected.");
```

Comment:
Grammatically, it's better to use the same tense: "did not appear, while it was expected"

Reply:
Good point! It is copy-paste from another test module. Updated.